tests/slr_integration/Orders/Controller_Test.php

```diff
@@ -581,4 +581,45 @@ public function test_my_tickets_page_has_seat_info( Closure $fixture ): void {
 		
 		$this->assertMatchesHtmlSnapshot( $html );
 	}
+	
+	public function tests_seats_row_actions_added() {
+		$this->set_fn_return( 'is_admin', true );
+		$this->make_controller()->register();
+		
+		$event_id = tribe_events()->set_args(
+			[
+				'title'      => 'Event with single seated attendee',
+				'status'     => 'publish',
+				'start_date' => '2020-01-01 00:00:00',
+				'duration'   => 2 * HOUR_IN_SECONDS,
+			]
+		)->create()->ID;
+		
+		update_post_meta( $event_id, Meta::META_KEY_ENABLED, true );
+		update_post_meta( $event_id, Meta::META_KEY_LAYOUT_ID, 'layout-id' );
+		
+		$ticket_id = $this->create_tc_ticket( $event_id, 10 );
+		
+		update_post_meta( $ticket_id, Meta::META_KEY_ENABLED, true );
+		update_post_meta( $ticket_id, Meta::META_KEY_LAYOUT_ID, 'layout-id' );
+		
+		$order = $this->create_order(
+			[ $ticket_id => 1 ],
+			[
+				'purchaser_email' => 'test-purchaser@test.com',
+			]
+		);
+		global $post;
+		$post = get_post( $event_id );
+		
+		$row_actions = apply_filters( 'post_row_actions', [], $post );
+		
+		$this->assertContains( 'tickets_seats', array_keys( $row_actions ) );
+		
+		// Removing the layout enabled meta should remove the row action.
+		delete_post_meta( $event_id, Meta::META_KEY_ENABLED );
+		
+		$row_actions = apply_filters( 'post_row_actions', [], $post );
```

Comment:
Use a JSON ssnapshot on the row actions, make them a JSON object and use the `JSON_SNAPSHOT_OPTIONS` for the format [link](https://github.com/the-events-calendar/event-tickets/blob/48fc217f70174db943fa764e1fcbefe8d6df4fb6/tests/slr_integration/_bootstrap.php#L32)